docs/install.rst

```diff
@@ -22,6 +22,16 @@ Distribution                         Installing
                                         # pacaur -S streamlink-git
 
                                      `Installing AUR packages`_
+`Debian (sid, testing)`_             .. code-block:: console
+
+                                        # apt update
+                                        # apt install streamlink
+`Debian (stable)`_                   .. code-block:: console
+
+                                        # wget -qO - "https://bintray.com/user/downloadSubjectPublicKey?username=amurzeau" | apt-key
```

Comment:
Isn't `apt-key` missing a few params here? Doesn't it also need root access?
```bash
wget -qO- "https://bintray.com/user/downloadSubjectPublicKey?username=amurzeau" | sudo apt-key add -
```

Reply:
Yes you are right about params, I used what I had on bintray description but it is wrong here too.
Ok about `sudo`, I was expecting `#` to mean "run as root" as opposed to "$"